src/main/java/com/synopsys/arc/jenkins/plugins/ownership/OwnershipPlugin.java

```diff
@@ -258,7 +258,11 @@ public String resolveEmail(User user) {
                 } else {
                     Class<MailAddressResolver> resolverClass = (Class<MailAddressResolver>)Class.forName(mailResolverClassName);
                     MailAddressResolver res = MailAddressResolver.all().get(resolverClass);
-                    return res.findMailAddressFor(user);
+                    if (res != null) {
+                        return res.findMailAddressFor(user);
+                    } else { 
+                        return null; 
```

Comment:
I understand how it could happen, but it rather an unrealistic use-case. But still fine.
The only thing is that returning `null` in this case is wrong, the code should fall through and use the default resolver like it happens for the missing class. I will add some logging in the follow-up PR